src/Uno.UI/UI/Xaml/Controls/MediaPlayerElement/MediaTransportControls.cs

```diff
@@ -994,89 +996,94 @@ private void UpdateMediaControlState(DependencyProperty property)
 					break;
 
 				case var _ when property == IsRepeatButtonVisibleProperty:
-					BindVisibility(m_tpRepeatButton, IsRepeatButtonVisible);
+					BindVisibility(m_tpRepeatButton, IsImplemented(typeof(MediaPlayer), "IsLoopingEnabled") && IsRepeatButtonVisible, property);
 					break;
 				case var _ when property == IsRepeatEnabledProperty:
 					BindIsEnabled(m_tpRepeatButton, IsRepeatEnabled);
 					break;
 				case var _ when property == IsVolumeButtonVisibleProperty:
-					BindVisibility(m_tpTHVolumeButton, IsVolumeButtonVisible);
+					BindVisibility(m_tpTHVolumeButton, IsImplemented(typeof(MediaPlayer), "Volume") && IsVolumeButtonVisible, property);
```

Comment:
Is Volume implemented in all platforms?

Reply:
I double checked and indeed I had looked in the wrong class.